### ISSUE TITLE
`test_thin_stack.py` is failing

### ISSUE THREAD

**Issue by hvy** [chainer/chainer#7100]
The following test file in the sentiment analysis example fails. https://github.com/chainer/chainer/blob/master/examples/sentiment/test_thin_stack.py

It is a part of an example and is thus not tested by the CI. I have not yet investigated the cause nor the effects but the loss on the training data seems to decrease.

**Comment by hvy** [comment]
We briefly discussed that it might be worth considering running theses tests in the CI environments.
